packages/searchable-dropdown/src/provider/controller.ts

```diff
@@ -109,7 +109,8 @@ export class SearchableDropdownController implements ReactiveController {
    * Close dropdown when click oustside host id
    */
   private _handleGlobalClick = (e: MouseEvent): void => {
-    if (e.target && (e.target as HTMLElement).id !== this.#host.id) {
+    const t = e.target as HTMLElement;
+    if (t && t !== (this.#host as unknown as HTMLElement)) {
```

Comment:
I see the intention, but can we guarantee that `this.#host` is a HTMLElement? According to the type of `this.#host` (`SearchableDropdownControllerHost`) it's not required. Should `SearchableDropdownControllerHost` type extend type `HTMLElement`?

Reply:
I tried to extend `SearchableDropdownControllerHost` with `HTMLElement`, but they have the conflicting property `autofocus`. Extending EventTarget does the trick.

Reply:
And `this.#host` will always be a HTMLElement in the event handler.